API/endpoints/Meeting.js

```diff
@@ -258,17 +258,41 @@ Meeting.addAltHosts.scopes = [
  * @method addAltHost
  * @param {object} options - object containing all arguments
  * @param {number} options.meetingId - the id for the meeting to add hosts to
- * @param {string} options.altHost - the emails or ids for the alt hosts to
- *   add
- * @returns {string} status where allowed values
- *   are: 'success' (added or already in list), 'no_user' (user was not found)
- *   other errors are thrown as ZACCLErrors
+ * @param {string} options.altHost - the email or id of the alt host to
+ *   be added
+ * @param {object} [options.meetingObj] - Zoom meeting object that needs to be
+ *   updated. Meeting ID is not used to fetch meeting if this object is passed
+ * @returns {Meeting} updated meeting object after adding alternative host
  */
 Meeting.addAltHost = function (options) {
-  // 1. Call addAltHosts with an array of length 1
-  // 2. Extract the status and return that
+  // Destructure arguments
+  const {
+    meetingId,
+    altHost,
+  } = options;
+
+  let { meetingObj } = options;
+
+  // Only call Meeting.get if no meetingObj passed
+  if (!meetingObj) {
+    meetingObj = this.api.meeting.get({ meetingId });
+  }
+
+  // extract altHosts into an array
+  const altHosts = meetingObj.settings.alternative_hosts.split(',');
+
+  // add as altHost only if not already present
+  if (!altHosts.includes(altHost)) {
+    meetingObj.settings.alternative_hosts = (
+      `${meetingObj.settings.alternative_hosts}, ${altHost}`
+    );
```

Comment:
This doesn't work if there aren't other alt-hosts. You'd end up with ", <email>"

Could you instead do this:

```
altHosts.push(altHost);
meetingObj.settings.alternative_hosts = altHosts.join(',');
```